scripts/get_latest_release.sh

```diff
@@ -1,14 +1,20 @@
 #!/usr/bin/env bash
-# Time: 2020-01-29 14:49:24
-# Author: https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
-get_latest_release() {
-  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
-    grep '"tag_name":' |                                            # Get tag line
-    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
-}
+# $1 - repo to check against
+# $2 - release branch pattern to match
```

Comment:
I personally would reassign the the variable to not have to look it up further down in the code as a readability improvement.

Reply:
This is the old  version - below is the newest, but I can get behind renaming the parameters to make it clearer

Reply:
Fixed - good catch for clarity - thank you!

> I personally would reassign the variable to not have to look it up further down in the code as a readability improvement.